src/components/Inventory/view/WithdrawalModal.tsx

```diff
@@ -199,31 +211,111 @@ export default function WithdrawalsModal(props: WithdrawalsModalProps): JSX.Elem
               </FormControl>
             </Grid>
 
-            {record.batchWithdrawals.map((bw, index) => {
-              return (
-                <Grid item xs={gridSize()} sx={marginTop} paddingRight={paddingSeparator} key={`batch-${index}`}>
-                  <Textfield
-                    id='readyQuantityWithdrawn'
-                    value={bw.readyQuantityWithdrawn}
-                    onChange={(id, value) => onChangeQuantity('readyQuantityWithdrawn', value)}
-                    type='text'
-                    label={strings.WITHDRAW_QUANTITY_REQUIRED}
-                    errorText={validateFields && bw.readyQuantityWithdrawn === 0 ? strings.REQUIRED_FIELD : ''}
+            {record.purpose === 'Out Plant' && (
+              <>
+                {record.batchWithdrawals.map((bw, index) => {
+                  return (
+                    <Grid item xs={gridSize()} sx={marginTop} paddingRight={paddingSeparator} key={`batch-${index}`}>
+                      <Textfield
+                        id='readyQuantityWithdrawn'
+                        value={bw.readyQuantityWithdrawn}
+                        onChange={(id, value) => onChangeQuantity('readyQuantityWithdrawn', value)}
+                        type='text'
+                        label={strings.WITHDRAW_QUANTITY_REQUIRED}
+                        errorText={validateFields && bw.readyQuantityWithdrawn === 0 ? strings.REQUIRED_FIELD : ''}
+                      />
+                    </Grid>
+                  );
+                })}
+
+                <Grid item xs={gridSize()} sx={marginTop} paddingLeft={paddingSeparator}>
+                  <DatePicker
+                    id='withdrawnDate'
+                    label={strings.WITHDRAW_DATE_REQUIRED}
+                    aria-label={strings.WITHDRAW_DATE_REQUIRED}
+                    value={record.withdrawnDate}
+                    onChange={changeDate}
+                    errorText={validateFields && !record.withdrawnDate ? strings.REQUIRED_FIELD : ''}
```

Comment:
withdrawn date can be pulled out from both conditions? common to all withdrawal purposes